### PR TITLE
Add ckeditor column width plugin

### PR DESCRIPTION
**Pull request by montgomeryyoung**
## Summary / Approach

Add a custom module that will enable the ckeditor column widths plugin.

## Testing Instruction(s)

Edit a table in ckeditor. Hover the mouse over a column bar and the cursor should change to indicate it is resizable.

## Metadata

| Question | Answer |
|----------|--------|
| Did you apply meaningful labels to the pull request? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests cover changes? | no
| Did you perform browser testing? | yes
| Did you provide detail in the summary on how and where to test this branch? | yes
| Risk level | low
| Relevant links | [RIGA-467](https://thinkoomph.jira.com/browse/RIGA-467)

### CODE REVIEW

File: ecms_base/modules/custom/ecms_ckeditor/ecms_ckeditor.info.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: eCMS CKEditor
+description: Custom CKEditor5 functionality for eCMS.
+type: module
+core_version_requirement: ^9 || ^10
```
Comment (pfrilling) [inline ecms_base/modules/custom/ecms_ckeditor/ecms_ckeditor.info.yml:4]: Might as well add 11 support now.

File: ecms_base/ecms_base.install [hunk 2]
```diff
@@ -2132,3 +2132,16 @@ function ecms_base_update_10200(array &$sandbox): void {
     $active_storage->write("{$config}", $install_source->read("{$config}"));
   }
 }
+
+/**
+ * Add ecms_ckeditor to the installed modules.
+ */
+function ecms_base_update_10201(array &$sandbox): void {
```
Comment (pfrilling) [inline ecms_base/ecms_base.install:2139]: Just a note that 10201 might have already been used in another update for this release. Might have a merge conflict at some point.

File: ecms_base/modules/custom/ecms_ckeditor/ecms_ckeditor.ckeditor5.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+ecms_ckeditor_tableColumnResize:
+  ckeditor5:
+    plugins:
+      - table.TableColumnResize
```
Comment (pfrilling) [inline ecms_base/modules/custom/ecms_ckeditor/ecms_ckeditor.ckeditor5.yml:4]: Is there any javascript that needs included?

**Review by pfrilling — CHANGES_REQUESTED** [review]
A few notes/questions.

File: ecms_base/modules/custom/ecms_ckeditor/ecms_ckeditor.ckeditor5.yml (hunk 3 above)
Comment (montgomeryyoung) [reply ecms_base/modules/custom/ecms_ckeditor/ecms_ckeditor.ckeditor5.yml:4]: no.

**Review by pfrilling — APPROVED** [review]

**Pull request merged**